src/bitcoin/bitcoinCoreRpc.ts

```diff
@@ -89,26 +89,29 @@ export class BitcoinCoreRpc implements BitcoinBlockchain {
 
   public async broadcastTransaction(transaction: Transaction): Promise<string> {
     const hex = transaction.toHex();
-    log("Broadcasting transaction ", hex);
+    logger.info("Broadcasting transaction ", hex);
     return this.bitcoinClient.sendRawTransaction(hex);
   }
 
   public async findHdOutputs(extendedPublicKeys: string[]): Promise<Utxo[]> {
     const scanobjects = extendedPublicKeys.map(exPubKey => {
-      log(`Send ${exPubKey} to bitcoind for scanning`);
+      logger.info(`Send ${exPubKey} to bitcoind for scanning`);
       return {
         desc: `combo(${exPubKey}/*)`,
         range: 1000
       };
     });
 
-    log("Starting `scantxoutset` which is a long blocking non-cached call");
+    logger.debug(
```

Comment:
🙃 Program flow logging

```suggestion
    logger.trace(
```

Reply:
I agreed with @D4nte that blockchain related logging should be on `info`.